src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Controllers/DataController.cs

```diff
@@ -545,8 +553,16 @@ private async Task<ActionResult> PutFormData(string org, string app, Instance in
 
             string serviceModelJsonString = JsonSerializer.Serialize(serviceModel);
 
-            // Trigger application business logic
-            bool changedByCalculation = await _altinnApp.RunCalculation(serviceModel);
+            bool changedByCalculation = false;
```

Comment:
Maybe we should add some sort of comment that we can easily search for later when we introduce these kinds of try/catch blocks?
For example `// Breaking change pending - version X`
That way when we make the major change we can search for all these comments and make sure we factor support for legacy.

Reply:
If you do, you can use a custom token ref. https://docs.microsoft.com/en-us/visualstudio/ide/using-the-task-list?view=vs-2019

If possible methods should be decorated with the [Obsolete] attribute.

Reply:
I guess this would be a `// HACK `then? As it is a temporary work around until next release?
Maybe we could even create an issue and use `// UNDONE #issuenumber` ?